content/automate/ManageIQ/Transformation/Infrastructure/VM/Common.class/__methods__/checkpoweredon.rb

```diff
@@ -7,20 +7,20 @@ module Common
             class CheckPoweredOn
               def initialize(handle = $evm)
                 @handle = handle
+                @task = ManageIQ::Automate::Transformation::Common::Utils.task(@handle)
```

Comment:
@fdupont-redhat This block of code is being used in 2 places it might make sense to move them into an embedded method in a future PR